GVFS/GVFS.Platform.POSIX/POSIXGitInstallation.cs

```diff
@@ -28,5 +29,17 @@ public string GetInstalledGitBinPath()
 
             return null;
         }
+
+        public string GetGVFSHooksPath()
+        {
+            // TODO(Mac): Use 'which' to find GVFS.Hooks (like the Windows platform uses "where")
+            string binPath = $"/usr/local/vfsforgit/{GVFSPlatform.Instance.Constants.GVFSHooksExecutableName}";

```

Comment:
I thought I saw a comment about this somewhere else, but can we just make this change now?

It might be possible to share the code for calling `which` that was added in #999

Reply:
On the Mac `which` is not able to find `GVFS.Hooks.exe` today - 
```
$ which gvfs.hooks
-bash: type: gvfs.hooks: not found
```

This is because it does not exist in the standard search paths. We can discuss the installer options to enable `which` (may be install a link like we do already for `gvfs`, or update `$PATH`). This is not directly related to this PR (service porting and auto-mount on Mac). I have created an [issue](https://github.com/Microsoft/VFSForGit/issues/1028) to track this. We can do this in a separate PR.